app/api/stage-configuration.js

```diff
@@ -0,0 +1,20 @@
+const Wreck = require('@hapi/wreck')
+const { applicationApiUri } = require('../config')
+
+async function getAllStageConfigurations () {
+  const url = `${applicationApiUri}/stageconfiguration`
+  try {
+    const response = await Wreck.get(url, { json: true })
+    if (response.res.statusCode !== 200) {
+      return null
```

Comment:
could be better to do something like this instead:

<img width="671" alt="image" src="https://github.com/DEFRA/ffc-ahwr-backoffice/assets/118533775/70ae8f85-2035-47eb-afd1-2d73f47614fc">

then the error handling/logging will be in one place (in catch block)